client/resource_group/controller/controller.go

```diff
@@ -386,8 +386,7 @@ func (c *ResourceGroupsController) Start(ctx context.Context) {
 				}
 
 			case gc := <-c.tokenBucketUpdateChan:
-				now := gc.run.now
-				go gc.handleTokenBucketUpdateEvent(c.loopCtx, now)
+				go gc.handleTokenBucketUpdateEvent(c.loopCtx)
```

Comment:
`now` is not used, we need to confirm if it is expected.

Reply:
It's okay to remove it